roles/openshift_health_checker/test/docker_image_availability_test.py

```diff
@@ -53,6 +53,7 @@ def execute_module(module_name, module_args, task_vars):
         ),
         openshift_deployment_type='origin',
         openshift_image_tag='3.4',
+        group_names=['nodes', 'masters'],
```

Comment:
I have one test case with etcd. I can add others for giggles but those are the only three that get containerized images [AFAICS](https://docs.openshift.org/latest/install_config/install/rpm_vs_containerized.html#install-config-install-install-methods-containerized).